example.py

```diff
@@ -1,27 +1,28 @@
 import os
 from dataclasses import dataclass
-from typing import Dict, List, Optional
+from typing import Dict, Generator, List, Optional
 
-from cognite.client.data_classes import Event
+from cognite.client.data_classes import Row
 
 from cognite.extractorutils.rest import RestExtractor
+from cognite.extractorutils.rest.types import Event, RawRow
 
 
 @dataclass
 class RawEvent:
```

Comment:
I'm not quite sure why we need to do this?

Reply:
Couple of reasons (notice this is the example file)
 * To show a slightly more realistic example where the source data model is not the destination data model directly
 * Easy workaround for casing changes (we should handle that in the library at some point)
 * `dacite`, which is the best turn-a-dict-to-a-class-instance library I know of, can only serialize `@dataclass`es, which the `Event` class from the SDK is not

Reply:
I see. Sounds reasonable!